content/django-vs-flask.md

```diff
@@ -0,0 +1,67 @@
+# Django or Flask which one is should you choose your project?
```

Comment:
@andrea-fajardo This is one of the topics James suggested to us. The topic is "Django vs. Flask", what title should we use to improve the SEO? Maybe "Django vs. Flask: Which one should you choose for your project?"